wp-content/themes/genesis-starter/gulp/tasks/images.js

```diff
@@ -10,7 +10,7 @@ const filterRetina = filter('**/*@2x.*', {
   restore: true
 });
 
-const filterSvg = filter('**/*.svg', {
+const filterSvgSprite = filter('svg-sprite/**/*.svg', {
```

Comment:
That he is, and a change he will have. Served with a siding of virgin... oil :octopus: